src/ontology/uberon-edit.obo

```diff
@@ -146056,10 +146057,10 @@ name: rhinarium
 def: "Moist, hairless, crenelated area around nostrils of some mammals." [http://anthro.palomar.edu/primate/glossary.htm, Wikipedia:Rhinarium]
```

Comment:
invalid ref: http://anthro.palomar.edu/primate/glossary.htm

Reply:
given that there is a wiki xref, will just remove this